assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -354,7 +354,11 @@ che.infra.kubernetes.ingress.domain=
 che.infra.kubernetes.namespace=
 
 # Defines Kubernetes Service Account name which should be specified to be bound to all workspaces pods.
-# Note that Che Server won't create the service account and it should exist.
+# Note that Kubernetes Infrastructure won't create the service account and it should exist.
```

Comment:
This line is a little unclear to me in related to `KubernetesServiceAccount.java` -- we're creating a SA there if necessary right?